From bf63e35c3036e6040c8cfecabc7160b1f36b0591 Mon Sep 17 00:00:00 2001
From: Randy MacLeod <Randy.MacLeod@windriver.com>
Date: Wed, 28 Aug 2019 12:31:15 -0400
Subject: [PATCH] adjust path filter for 2 memcheck tests

Test executables produced when cross-compiling can contain
relative paths such as:
   coregrind/tests/../../../valgrind-3.15.0/coregrind/
Use the --fullpath-after option to match and therefore
suppress more of the prefix to enable test to pass.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Randy MacLeod <Randy.MacLeod@windriver.com>
---
 memcheck/tests/badfree3.vgtest | 2 +-
 memcheck/tests/varinfo5.vgtest | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/memcheck/tests/badfree3.vgtest b/memcheck/tests/badfree3.vgtest
index 3dfc5fd8a..57eec21f3 100644
--- a/memcheck/tests/badfree3.vgtest
+++ b/memcheck/tests/badfree3.vgtest
@@ -1,3 +1,3 @@
 prog: badfree
-vgopts: -q --fullpath-after=memcheck/ --fullpath-after=coregrind/
+vgopts: -q --fullpath-after=/valgrind-3.15.0/memcheck/ --fullpath-after=/valgrind-3.15.0/coregrind/
 stderr_filter_args: badfree.c
diff --git a/memcheck/tests/varinfo5.vgtest b/memcheck/tests/varinfo5.vgtest
index 063d00dce..6907bb2f6 100644
--- a/memcheck/tests/varinfo5.vgtest
+++ b/memcheck/tests/varinfo5.vgtest
@@ -1,3 +1,3 @@
 prog: varinfo5
-vgopts: --fullpath-after=memcheck/  --fullpath-after=coregrind/ --read-var-info=yes --read-inline-info=yes -q
+vgopts: --fullpath-after=/valgrind-3.15.0/memcheck/  --fullpath-after=/valgrind-3.15.0/coregrind/ --read-var-info=yes --read-inline-info=yes -q
 stderr_filter: filter_varinfo3
-- 
2.22.0

